config/samples/simplekafkacluster_affinity.yaml

```diff
@@ -0,0 +1,263 @@
+apiVersion: kafka.banzaicloud.io/v1beta1
+kind: KafkaCluster
+metadata:
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: kafka
+spec:
+  headlessServiceEnabled: true
+  zkAddresses:
+    - "zookeeper-client.zookeeper:2181"
+  propagateLabels: false
+  oneBrokerPerNode: false
+  clusterImage: "ghcr.io/banzaicloud/kafka:2.13-2.6.0-bzc.1"
+  readOnlyConfig: |
+    auto.create.topics.enable=false
+    cruise.control.metrics.topic.auto.create=true
+    cruise.control.metrics.topic.num.partitions=1
+    cruise.control.metrics.topic.replication.factor=2
+  brokerConfigGroups:
+    default:
+      affinity:
+        podAntiAffinity:
```

Comment:
Add some comment to this example explaining what is achieved with this pod anti-affinity.